ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRow.java

```diff
@@ -38,6 +48,11 @@ public static PubsubMessageToTableRow of(ValueProvider<String> tableSpecTemplate
   }
 
   private final ValueProvider<String> tableSpecTemplate;
+  private final Cache<TableId, Boolean> tableExistenceCache = CacheBuilder.newBuilder()
+      .expireAfterWrite(Duration.ofMinutes(1)).build();
```

Comment:
is this relevant to here: (does the cache not get serialized and this value goes null outside testing?) https://github.com/mozilla/gcp-ingestion/blob/master/ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParsePayload.java#L52-L54

Reply:
That's possible. Good call. Will follow up with a PR to make this also a transient field we initialize on first use.